tox.ini

```diff
@@ -21,6 +21,8 @@ commands =
 [testenv:build_docs]
 basepython = python3.7
 extras = all,docs
+deps =
```

Comment:
I would add zeep here to see if that fixes it.

Reply:
It created another dependency issue

Reply:
Yeah, add the next one and hopefully it will stop.